README.md

```diff
@@ -169,17 +169,18 @@ views:
-| optional                     | Allows to have a column specified in render-table that is actually not present.                                                                                                                                                                | false   | true, false               |
-| display-mode                 | Allows to hide columns from views by setting this to `hidden` or have a column only in [detail view](https://examples.bootstrap-table.com/#options/detail-view.html#view-source) by setting this to `detail`.                                  | normal  | detail, normal, hidden    |
-| precision                    | Allows to specify the precision of floats. It expects an integer specifying the decimal places that will be shown. Values smaller than $1 / (10^{precision})$ will be displayed in scientific notation with the same number of decimal places. | 2       |                           |
-| plot-view-legend             | Specifies whether the column in the plot-view should include a legend or not.                                                                                                                                                                  | false   | true, false               |
+| keyword                     | explanation                                                                                                                                                                                                                                    | default | possible values           |
+|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|---------------------------|
+| link-to-url                 | Renders a link to the given url with {value} replace by the value of the table. Other values of the same row can be accessed by their column header (e.g. {age} for a column named age).                                                       |         |                           |
+| custom                      | Applies the given js function to render column content. The parameters of the function are similar to the ones defined [here](https://bootstrap-table.com/docs/api/column-options/#formatter)                                                  |         |                           |
+| custom-path                 | Allows to specify a path to a file that contains a js function similar to custom.                                                                                                                                                              |         |                           |
```

Comment:
We should say something about the naming of the function and in general the allowed contents of the file here.

Reply:
The name of the function shouldn't matter (added that to the docs) and also pointed to the example of our example report.